src/assets/i18n/en.json5

```
@@ -862,6 +862,12 @@
 
   "collection.edit.tabs.authorizations.title": "Collection Edit - Authorizations",
 
+  "collection.edit.item.authorizations.load-bundle-button": "Load more bundles",
+
+  "collection.edit.item.authorizations.load-more-button": "Load more",
+
+  "collection.edit.item.authorizations.show-bitstreams-button": "Show all Bitstreams' Policies for Bundle",
```

I'd simplify this to 
```suggestion
  "collection.edit.item.authorizations.show-bitstreams-button": "Show bitstream policies for this bundle",
```

---

I like @artlowel 's suggestion, though it looks like this is the button text where the Bundle name is appended on the end.  So, it likely should be:
`Show bitstream policies for bundle`

(I just removed "this" because otherwise it'd say "Show bitstream policies for this bundle LICENSE" or similar)